edrumulus_parameters.cpp

```diff
@@ -57,6 +57,91 @@ void Edrumulus::Pad::apply_preset_pad_settings()
+      break;
+
+    case LEHHS12C_CTRL: // 0-35k poti
+      pad_settings.is_control              = true;
+      pad_settings.pos_threshold           = 6;
+      pad_settings.pos_sensitivity         = 23;
+      pad_settings.velocity_threshold      = 0;
+      pad_settings.velocity_sensitivity    = 22;
+      break;
+
+    case MPS750X_TOM: // dual trigger
+      pad_settings.scan_time_ms            = 6.0f;
+      pad_settings.pre_scan_time_ms        = 3.5f;
+      pad_settings.decay_grad_fact2        = 150.0f;
+      pad_settings.decay_len2_ms           = 450.0f;
+      pad_settings.rim_shot_is_used        = false;
+      pad_settings.velocity_threshold      = 2;
+      pad_settings.velocity_sensitivity    = 3;
+      break;
+
+    case MPS750X_SNARE: // dual trigger
+      pad_settings.scan_time_ms            = 6.0f;
+      pad_settings.pre_scan_time_ms        = 3.5f;
+      pad_settings.decay_grad_fact2        = 150.0f;
+      pad_settings.decay_len2_ms           = 450.0f;
+      pad_settings.rim_use_low_freq_bp     = false;
+      pad_settings.rim_shot_is_used        = true;
+      pad_settings.rim_shot_treshold       = 12;
+      pad_settings.velocity_threshold      = 2;
+      pad_settings.velocity_sensitivity    = 6;
+      pad_settings.pos_sense_is_used       = true;
+      pad_settings.pos_sensitivity         = 3;
+      break;
+
+    case MPS750X_KICK: // single trigger
+      pad_settings.first_peak_diff_thresh_db = 3.0f;
+      pad_settings.scan_time_ms              = 6.0f;
+      pad_settings.pre_scan_time_ms          = 3.5f;
+      pad_settings.decay_grad_fact2          = 150.0f;
+      pad_settings.decay_len2_ms             = 450.0f;
+      pad_settings.velocity_threshold        = 10;
+      pad_settings.velocity_sensitivity      = 10;
+      pad_settings.curve_type                = LOG2;
+      break;
+
+    case MPS750X_RIDE: // 3-zone
+      pad_settings.is_rim_switch             = true;
+      pad_settings.scan_time_ms              = 3.0f;
+      pad_settings.decay_est_delay_ms        = 1.0f;
+      pad_settings.decay_grad_fact2          = 150.0f;
+      pad_settings.decay_len2_ms             = 450.0f;
+      pad_settings.rim_use_low_freq_bp       = false;
+      pad_settings.rim_shot_is_used          = true;
+      pad_settings.rim_shot_treshold         = 18;
+      pad_settings.rim_shot_boost            = 0;
+      pad_settings.velocity_threshold        = 12;
+      pad_settings.velocity_sensitivity      = 6;
+      break;
+
+    case MPS750X_CRASH: // 2-zone
+      pad_settings.is_rim_switch             = true;
+      pad_settings.scan_time_ms              = 3.0f;
+      pad_settings.decay_est_delay_ms        = 1.0f;
+      pad_settings.decay_grad_fact2          = 150.0f;
+      pad_settings.decay_len2_ms             = 450.0f;
+      pad_settings.rim_use_low_freq_bp       = false;
+      pad_settings.rim_shot_is_used          = true;
+      pad_settings.rim_shot_treshold         = 20;
```

Comment:
I fixed the typo treshold on main. To make it compatible, you simply have to add the missing "h" at these places.